microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -520,40 +631,41 @@ def verify_azure_file_share(
         xfstests = self._install_xfstests(node)
 
         azure_file_share = node.features[AzureFileShare]
-        version = azure_file_share.get_smb_version()
-        mount_opts = (
-            f"-o vers={version},credentials=/etc/smbcredentials/lisa.cred"
-            ",dir_mode=0777,file_mode=0777,serverino"
-        )
-
         random_str = generate_random_chars(string.ascii_lowercase + string.digits, 10)
         file_share_name = f"lisa{random_str}fs"
         scratch_name = f"lisa{random_str}scratch"
-
-        # fs_url_dict: Dict[str, str] = {file_share_name: "", scratch_name: ""}
-        try:
-            fs_url_dict = azure_file_share.create_file_share(
-                file_share_names=[file_share_name, scratch_name],
-                environment=environment,
-            )
-            test_folders_share_dict = {
-                _test_folder: fs_url_dict[file_share_name],
-                _scratch_folder: fs_url_dict[scratch_name],
-            }
-            azure_file_share.create_fileshare_folders(test_folders_share_dict)
-
-            self._execute_xfstests(
-                log_path,
-                xfstests,
-                result,
-                test_dev=fs_url_dict[file_share_name],
-                scratch_dev=fs_url_dict[scratch_name],
-                excluded_tests=self.excluded_tests,
-                mount_opts=mount_opts,
-            )
-        finally:
-            # clean up resources after testing.
-            azure_file_share.delete_azure_fileshare([file_share_name, scratch_name])
```

Comment:
Keep the deletion part, it needs to recycle the resources and clean up for the VM.

Reply:
Please keep the finally part.